idunn/instant_answer/patterns_to_ignore.txt

```diff
@@ -0,0 +1,25 @@
+qwant ?maps?
+maps? qwant
+google ?maps?
+maps? google
+maps?( of)?
+carte( des?| du)?
+plan( de| du)?
+add?resse?
+((num(e|é)ro)|n(o|°)?)( de)? ?t(e|é)l(e|é)phone
+num(e|é)ro$
```

Comment:
Is `$` intentional? You may have an example in mind where it makes sense to keep this keyword?

```suggestion
num(e|é)ro
```

(same with `menu$` below)

Reply:
These tokens are mostly used at the end of queries, so this was a (superfluous ?) precaution for cases when a specific POI name would start with "numero ..." or "menu ...".